tests/browser/test/procedure_test.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Node.js script to run Automated tests in Chrome, via webdriver.
+ */
+
+const webdriverio = require('webdriverio');
+const chai = require('chai');
+const path = require('path');
+const {posixPath} = require('../../../scripts/helpers');
+
+let browser;
+suite('Testing Connecting Blocks', function (done) {
+  // Setting timeout to unlimited as the webdriver takes a longer time to run than most mocha test
+  this.timeout(0);
+
+  // Setup Selenium for all of the tests
+  suiteSetup(async function () {
```

Comment:
Is this setup going to be the same for every test file? Might be worth moving it into its own helper method (in a separate PR).

Reply:
Great call I can do that i follow up CL